apps/web/tailwind.config.js

```diff
@@ -12,11 +12,183 @@ module.exports = {
   ],
   theme: {
     colors: {
-      gray: '#1E2025',
-      darkgray: '#0F1012',
-      white: '#FFFFFF',
-      muted: '#8A919E',
-      yellow: '#FFD200',
+      'white': '#FFFFFF',
+      'blue-0': '#001033',
```

Comment:
Tailwind allows us to format this like:

```
'blue': {
   0: '#001033'.
 }
 ```
 
 etc, like the way translucent works, do U mind refactoring to follow that format?

Reply:
Thanks for the pointer! Got it fixed in the next commit.